src/main/java/com/bbaga/githubscheduledreminderapp/domain/configuration/sources/filters/LabelFilterConfig.java

```diff
@@ -14,15 +15,31 @@ public class LabelFilterConfig extends AbstractFilterConfig {
 
     public static final String LABEL_ACKNOWLEDGED = "acknowledged";
 
-    @JsonProperty("exclude-labels")
+    @JsonProperty("include")
+    private List<String> includeLabels;
+
+    @JsonProperty("exclude")
+    @JsonAlias("exclude-labels")
     private List<String> excludeLabels;
 
     @JsonProperty("expiry-days")
-    private int expiryDays = 90;
+    private int expiryDays = Integer.MAX_VALUE;
```

Comment:
```suggestion
    private int expiryDays = 90;
```

There are a number of users that may or may not have this value specified in their configuration file, I would like to keep this value at 90 days until there is a way to make users aware of changes in the defaults, or config versioning is introduced.

In the mean time, using `0` for the `expiry-days` value in the configuration should (or could) have a similar effect.

Reply:
alright. I sort of dont like that expiry-days is mandatory to enter... to not use it. 
But breaking current functionality is also bad. Will change this.